src/js/PhotoSphereViewer.core.js

```diff
@@ -74,8 +74,8 @@ PhotoSphereViewer.prototype._loadXMP = function() {
     throw new PSVError('Cannot load image');
   };
 
-  if (true === this.prop.cacheTextures && 'undefined' != typeof this.prop.loadedXMP[this.config.panorama]) {
-    defer.resolve(this.prop.loadedXMP[this.config.panorama]);
+  if (true === this.config.cacheTextures && 'undefined' != typeof this.config.loadedXMP[this.config.panorama]) {
+    defer.resolve(this.config.loadedXMP[this.config.panorama]);
```

Comment:
Don't want to bother you but THIS has it's place in "prop"... same for "loadedTextures".
It's all about separate concerns (configurable vs internal)